packages/sdk-cli/src/api/baseAPI.test.ts

```diff
@@ -23,6 +24,30 @@ describe('apiFetch()', () => {
     ).rejects.toThrowErrorMatchingSnapshot();
   });
 
+  it('can use a custom API URL and skip auth', async () => {
+    jest
+      .spyOn(fetchModule, 'default')
+      .mockImplementationOnce(
+        async (url: RequestInfo, init?: RequestInit) =>
+          ({ url, headers: init?.headers } as Response),
+      );
+
+    const fakeAPIDomain = 'https://fake-dev-relay-test-endpoint.fitbit.com';
+
+    mockAuthToken(null);
+    // apiFetch would throw if it didn't skip auth, because authToken is falsy
```

Comment:
you can actually check this, rather than awaiting it, use `await expect(response).resolves.toBeDefined()`

Reply:
if on the above line I assigned an `awaited` value to response, the response is no longer a Promise, right? So I can just do `expect(response).toBeDefined()`

Reply:
You can, but I think Jest gives a better error if you let it handle the resolution/rejection if it doesn't resolve as expected.

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/26ee1d9e19334aedfd889137a6271b563b290a65 ?

Reply:
Yep, looks fine.